packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -220,10 +221,13 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
       await touchFile(this.outputPath)
       await writeFile(this.outputPath, '(()=>{})();')
     }
+
+    await this.specification.postBuildAction?.(this)
```

Comment:
```suggestion
    await this.specification.postBuildAction?.(this, options.app)
```
The app is available here ☝️